osu.Game.Rulesets.Rush/RushInputManager.cs

```diff
@@ -29,10 +30,39 @@ public enum RushAction
         [Description("Ground (Secondary)")]
         GroundSecondary,
 
+        [Description("Ground (Tertiary)")]
+        GroundTertiary,
+
+        [Description("Ground (Quaternary)")]
+        GroundQuaternary,
+
         [Description("Air (Primary)")]
         AirPrimary,
 
         [Description("Air (Secondary)")]
-        AirSecondary
+        AirSecondary,
+
+        [Description("Air (Tertiary)")]
+        AirTertiary,
+
+        [Description("Air (Quaternary)")]
+        AirQuaternary
+    }
+
+    public static class RushActionExtensions
+    {
+        public static LanedHitLane Lane(this RushAction action) =>
+            action switch
+            {
+                RushAction.GroundPrimary => LanedHitLane.Ground,
+                RushAction.GroundSecondary => LanedHitLane.Ground,
+                RushAction.GroundTertiary => LanedHitLane.Ground,
+                RushAction.GroundQuaternary => LanedHitLane.Ground,
+                RushAction.AirPrimary => LanedHitLane.Air,
+                RushAction.AirSecondary => LanedHitLane.Air,
+                RushAction.AirTertiary => LanedHitLane.Air,
+                RushAction.AirQuaternary => LanedHitLane.Air,
+                _ => LanedHitLane.Ground
+            };
```

Comment:
```suggestion
        public static LanedHitLane Lane(this RushAction action) => action switch
        {
            RushAction.GroundPrimary => LanedHitLane.Ground,
            RushAction.GroundSecondary => LanedHitLane.Ground,
            RushAction.GroundTertiary => LanedHitLane.Ground,
            RushAction.GroundQuaternary => LanedHitLane.Ground,
            RushAction.AirPrimary => LanedHitLane.Air,
            RushAction.AirSecondary => LanedHitLane.Air,
            RushAction.AirTertiary => LanedHitLane.Air,
            RushAction.AirQuaternary => LanedHitLane.Air,
            _ => LanedHitLane.Ground
        };
```

Against this type of style, either put the `action switch` at the top or make it an actual method body, but the former sounds good for this case.

Reply:
I'd prefer it how it was personally, since I think the `=>` should always be at the end of the line if you're not doing a single line expression.  What you've suggested is fine, I guess.

Reply:
![image](https://user-images.githubusercontent.com/31809541/91665121-f267ea80-eab8-11ea-8842-39bfe8334dfc.png)
Method body would look something like this, which makes more sense to me imo.

Reply:
![image](https://user-images.githubusercontent.com/70440919/91665162-5069f180-eafc-11ea-9e30-72d9e6046541.png)
fix your spacing.